CMake/ITKModuleExternal.cmake

```diff
@@ -11,20 +11,8 @@ if(NOT EXISTS ${ITK_CMAKE_DIR}/ITKModuleMacros.cmake)
   message(FATAL_ERROR "Modules can only be built against an ITK build tree; they cannot be built against an ITK install tree.")
 endif()
 
-if(ITK_WRAP_PYTHON)
-  if(DEFINED Python3_EXECUTABLE)
-    set(_specified_Python3_EXECUTABLE ${Python3_EXECUTABLE})
-  endif()
-  find_package(Python3 COMPONENTS Interpreter Development)
-  set(Python3_EXECUTABLE ${_specified_Python3_EXECUTABLE} CACHE INTERNAL
-    "Path to the Python interpreter" FORCE)
-else()
-  find_package(Python3 COMPONENTS Interpreter)
-endif()
-if(NOT Python3_EXECUTABLE AND _Python3_EXECUTABLE)
-  set(Python3_EXECUTABLE ${_Python3_EXECUTABLE} CACHE INTERNAL
-    "Path to the Python interpreter" FORCE)
-endif()
+set(PYTHON_DEVELOPMENT_REQUIRED ${ITK_WRAP_PYTHON})
+include(${CMAKE_CURRENT_SOURCE_DIR}/ITKSetPython3Vars.cmake)
```

Comment:
Thanks. This does not seem to work when building an external module against an already built ITK, see [here](https://my.cdash.org/buildSummary.php?buildid=1789345). Replacing this line by ` include(ITKSetPython3Vars)` seems to help but a compilation is on-going on my side, I'll send you the dashboard link when it's done.

Reply:
I think that is a chicken and egg problem.  We need to update the ITK environment on the docker image that is used to build external modules.

Reply:
OH!! The path is also wrong!  I see that now.

Reply:
My compilation [works](https://my.cdash.org/buildSummary.php?buildid=1789429) with this change but I haven't checked the compilation from within ITK yet.

Reply:
@SimonRit I'm dealing with COVID-19 implications of all lectures going virtual, and ISBI 2020 becoming a completely virtual meeting.  I'd really appreciate your response if you feel that your tests are working, then I'll merge after that.  



Thanks,
Hans